tokio-codec/src/length_delimited.rs

```diff
@@ -15,8 +15,8 @@
 //! byte stream into a stream of frames.
 //!
 //! ```
-//! use tokio::io::{AsyncRead, AsyncWrite};
-//! use tokio::codec::*;
+//! use tokio_io::{AsyncRead, AsyncWrite};
+//! use tokio_codec::*;
```

Comment:
minor nit: even though this was copied can we update this to not use a glob import?

Reply:
Done